spyder/widgets/sourcecode/api/decorations.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
```

Comment:
@ccordoba12 the idea is that complex widgets will have some sort of api, this is the case of the editor. It would be very weird to decouple the api of the code editor and put it in a "global" spyder.api folder. The idea is for widgets to be self contained as all other spyder widgets.